src/OpenLoco/src/Windows/Options.cpp

```diff
@@ -42,6 +42,12 @@ namespace OpenLoco::Ui::Windows::Options
     static loco_global<ObjectManager::SelectedObjectsFlags*, 0x011364A0> __11364A0;
     static loco_global<uint16_t, 0x0112C185> _112C185;
 
+    // TODO: This shouldn't be required but its due to how the lifetime
+    // of the string needs to exist beyond a prepare draw function and
+    // into a draw function. Rework when custom formatter can store full
+    // strings or widgets can store dynamically allocated strings.
+    static std::string _chosenLanguage;
```

Comment:
So technically it looks like i didn't need to make this change but at the same time this is also an issue.